volatility3/framework/objects/utility.py

```diff
@@ -105,7 +105,7 @@ def pointer_to_string(
 
 def gather_contiguous_bytes_from_address(
```

Comment:
I think that's the wrong method to change the signature on?  I think you meant to change `bytes_to_decoded_string` which says it returns bytes...